src/ext-test/java/org/opentripplanner/ext/flex/ScheduledDeviatedTripTest.java

```diff
@@ -270,7 +270,7 @@ private static NearbyStop getNearbyStop(FlexTrip<?, ?> trip, String id) {
       stopLocation,
       0,
       List.of(),
-      new State(
+      State.createState(
```

Comment:
I think `State.createState` as a bit redundant. How do you like `State.create`?

Reply:
Done